driver/kubedriver_test.go

```diff
@@ -89,26 +125,27 @@ func TestKubeRun(t *testing.T) {
 		StringData: map[string]string{ExperimentPath: string(byteArray)},
 	}, metav1.CreateOptions{})
 
-	_, _ = kd.Clientset.BatchV1().Jobs("default").Create(context.TODO(), &batchv1.Job{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      "default-1-job",
-			Namespace: "default",
-			Annotations: map[string]string{
-				"iter8.tools/group":    "default",
-				"iter8.tools/revision": "1",
-			},
-		},
-	}, metav1.CreateOptions{})
-
-	err := base.RunExperiment(false, kd)
+	// _, _ = kd.Clientset.BatchV1().Jobs("default").Create(context.TODO(), &batchv1.Job{
+	// 	ObjectMeta: metav1.ObjectMeta{
+	// 		Name:      "default-1-job",
+	// 		Namespace: "default",
+	// 		Annotations: map[string]string{
+	// 			"iter8.tools/group":    "default",
+	// 			"iter8.tools/revision": "1",
+	// 		},
+	// 	},
+	// }, metav1.CreateOptions{})
```

Comment:
Is this no longer necessary? When I commented this section, tests still ran fine.

Reply:
I am unsure what role this plays in the test case.

Reply:
I agree. I am unsure the purpose of this.